Source/MSBuild.Sdk.Extras/Build/LanguageTargets/Xamarin.targets

```diff
@@ -34,4 +34,12 @@
     <Error Text="The Binding Project for $(_SdkTargetPlatformName) is only supported with only C#, not with $(_SdkLanguageSourceName). Please change the project to C# to use the Bindings Project.'"/>
   </Target>
 
+  <!-- A workaround for https://github.com/microsoft/msbuild/issues/4584 and resolves https://github.com/onovotny/MSBuildSdkExtras/issues/176 -->
+  <Target Name="_RemoveNativeReferencesManifest" Condition="'$(IsApplePlatform)' == 'true'" AfterTargets="BuiltProjectOutputGroup">
```

Comment:
I updated this to exclude the binding projects test because `@(NativeReference)` is also valid in non-binding projects.